content/en/install.md

```diff
@@ -3,101 +3,206 @@ title: Installation
+but lack some reproducibility benefits of project-based workflows.
+
+### Installing with `pip`
+
+1.  [Install Python](https://www.python.org/downloads/).
+
+2.  Create and activate a virtual environment with `venv`.
+
+    {{< admonition hint >}}
+See [the tutorial in the Python Packaging User Guide](https://packaging.python.org/en/latest/tutorials/installing-packages/#creating-virtual-environments).
+    {{< /admonition >}}
+
+3.  Install SciPy, using [`pip`]:
+
+        python -m pip install scipy
+
+[`pip`]: https://pip.pypa.io/en/stable/getting-started/
+
+### Installing with `conda`
+
+[Miniforge] is the recommended way to install `conda` and [`mamba`],
+two Conda-based environment managers.
+After creating an environment, you can install SciPy from conda-forge as follows:
+
+    conda install scipy # or
+    mamba install scipy
+
+[Miniforge]: https://conda-forge.org/download/
+[`mamba`]: https://mamba.readthedocs.io/en/latest/
+
+<a name="system-package-managers"></a>
+
+## Installing system-wide via a system package manager
 
 System package managers can install the most common Python packages.
 They install packages for the entire computer, often use older versions,
-and don't have as many available versions.
+and don't have as many available versions. They are not the recommended
+installation method.
 
-## Ubuntu and Debian
+### Ubuntu and Debian
 
 Using `apt-get`:
 
     sudo apt-get install python3-scipy
 
-## Fedora
+### Fedora
 
 Using `dnf`:
 
     sudo dnf install python3-scipy
 
-## macOS
+### macOS
 
 macOS doesn't have a preinstalled package manager, but you can install
 [Homebrew](https://brew.sh/) and use it to install SciPy (and Python itself):
 
     brew install scipy
 
-<a name="source"></a>
+<a name="building-from-source"></a>
 
-# Source packages
+## Building from source
 
 A word of warning: building SciPy from source can be a nontrivial exercise. We
-recommend using binaries instead if those are available for your platform.
+recommend using binaries instead if those are available for your platform
+via one of the above methods.
 For details on how to build from source, see
-[this guide in the SciPy docs](https://scipy.github.io/devdocs/building/index.html).
+[the building from source guide in the SciPy docs][building-docs].
+
+[building-docs]: https://scipy.github.io/devdocs/building/index.html
+
+<a name="type-stubs"></a>
+
+## Installing with type stubs
+
+Static type stubs are available via a separate package, `scipy-stubs`, on
+PyPI and conda-forge.
+You can also install SciPy and `scipy-stubs` as a single package,
+via the `scipy-stubs[scipy]` extra on PyPI, or the `scipy-typed`
+package on conda-forge.
+To get a specific version `x.y.z` of SciPy (such as `1.14.1`),
+you should install version `x.y.z.*`, for example:
+
+    uv add "scipy-stubs[scipy]==1.14.1.*" # or
+    pixi add "scipy-typed=1.15.0.*" # or
+    python -m pip install "scipy-stubs[scipy]" # or
+    conda install "scipy-typed>=1.14"
```

Comment:
This section reads very nicely now, btw, thanks!

In conda, does scipy-typed automatically match up with SciPy? If so, then this section is :ok_hand:

Reply:
> In conda, does scipy-typed automatically match up with SciPy?

Yep! The relevant lines from the recipe are https://github.com/conda-forge/scipy-stubs-feedstock/blob/fc110f760877e2b8cfe169cb9f54d308ffeb7bd9/recipe/meta.yaml#L38-L47 and https://github.com/conda-forge/scipy-stubs-feedstock/blob/fc110f760877e2b8cfe169cb9f54d308ffeb7bd9/recipe/meta.yaml#L1-L2.

Reply:
FYI, `scipy-stubs==1.15.0.0` is [almost ready](https://github.com/jorenham/scipy-stubs/issues/250), so I'll definitely be able to release it *before* the `scipy==1.15.0` final release.